aep/general/0005/aep.md

```diff
@@ -0,0 +1,84 @@
+# Designing an API
+
+This AEP serves as a high-level guide on designing an API that adheres to the
+AEPs.
+
+## Process summary
+
+1. enumerate the user case would like your API to satisfy.
+1. identify resources and add them to your API.
+1. identify standard operations and add them to your API.
+1. identify custom operations and add them to your API.
+
+## Enumerate use cases
+
+The first step in designing an API is understanding precisely what you would
+like your user to be able to accomplish your API.
+
+Enumerate one or more use cases to clarify what precisel
+
+For example:
+
+- Create a VM.
+- List all VMs owned by a company.
+- List all VMs owned by a user.
+- Restart a running VM.
+- Trade an in-game item with another player.
```

Comment:
I appreciate the love for Roblox...but since this example doesn't fit with the others, maybe drop it?

(Or have two separate sets of examples, where the second one might include this and also things like "list items in a player's inventory", "propose a trade", "accept a trade", "find open trade offers containing an item", etc.)

Reply:
The list wasn't intended to be an example of a user designing an API themselves... just examples of use cases.

I see why it's confusing to lump them together, but I also think distinct sets of use cases will help consumers of the AEPs understand the scope is all APIs, not just cloud / games / etc.

I split them up, PTAL.